build.sh

```diff
@@ -23,10 +28,17 @@ cd wls-module
 docker build -t wls:0.0.1 .
 cd ..
 
-cd analysis-module
-docker build -t analysis .
+cd ./monitoring/container-proc
+sh ./build.sh
 cd ..
 
+# Prepare container proc monitoring venv
+python3 -m venv ./cproc-venv --without-pip
+source ./cproc-venv/bin/activate   
+curl https://bootstrap.pypa.io/get-pip.py | python
+pip3 install -r ./monitoring/container-proc/requirements.txt
+deactivate
```

Comment:
Do we still need this venv? Since it is in a Docker image, shouldn't this be deleted?